audio/src/commandhandler.py

```diff
@@ -69,7 +66,7 @@ def _set_volume(self, entity_id, params):
         sound_id = self._sounds.get(entity_id)
         if not sound_id:
             return self._create_error_msg("Audio not found. Did you add it?")
-        if params.get("volume_left") or params.get("volume_right"):
+        if not params.get("volume_left") is None or not params.get("volume_right") is None:
```

Comment:
I had trouble parsing this because of the order of operations. Use "X is not None" instead of "not X is None"?

Or the more direct:
if "volume_left" in params or "volume_right" in params:

Reply:
Will fix!